### ISSUE TITLE
Wrong use of {box_name} in strings

### ISSUE THREAD

**Issue by ikmaak** [freedombox/Plinth#431]
There are some strings where FreedomBox was changed in {box_name} where it would be less appropriate to do so:

plinth/modules/help/templates/help_about.html:30
plinth/modules/help/templates/help_about.html:56
plinth/modules/help/templates/help_about.html:64
plinth/modules/help/templates/help_index.html:29
plinth/modules/help/templates/help_index.html:36
plinth/modules/help/templates/help_index.html:43
plinth/modules/help/templates/help_index.html:53
plinth/modules/first_boot/forms.py:116

I think that in these cases FreedomBox should still be used, as these do not talk about this box, but FreedomBox in general.

**Comment by jvalleroy** [comment]
The idea for using {box_name} is that someone could completely re-brand the interface. For example, someone could modify the software and call it "AwesomeBox", and that's ok, but we want to distinguish it from FreedomBox (i.e. so we don't get bug reports for their modified software).

This feature is not complete though. We still need to support customizing the logo, mailing list, wiki, IRC, pagekite voucher server, etc.

**Comment by ikmaak** [comment]
Then I would advise {project_name} for more clarity, and differentiation from the box instance name.

**Comment by jvalleroy** [comment]
I agree we should change the name to be more clear, but I would suggest "product_name".

**Comment by ikmaak** [comment]
Personally I like the title "project" better instead of the more derogatory term "product". It reduces free users to consumers.

**Comment by mridulnagpal** [comment]
I would like to work on this issue. What should I change box_name to ?

**Comment by jvalleroy** [comment]
@mridulnagpal We can use "project_name".

**Comment by mridulnagpal** [comment]
@jvalleroy Do I change all the occurences of box_name to project_name ?

**Comment by jvalleroy** [comment]
I guess the question is:
- Do we always want to refer to the project that produced the software ("FreedomBox")?
- Is there any case where we might want to refer to the box instance, using a different name?

**Comment by harry-7** [comment]
@mridulnagpal  are you still working on this ?

**Comment by mridulnagpal** [comment]
No this issue is free right now, you can surely take this up

**Comment by SunilMohanAdapa** [comment]
@ikmaak @jvalleroy , I believe using `product_name` (or like Mozilla `brand_name`) is more appropriate.

The one reason, why people would want to change the name `FreedomBox` is when there is branding to do.  Consider a hardware vendor wanting to ship FreedomBox but with their own brand.  This could happen because they don't have trademark permissions from the foundation.  This is very similar to how Mozilla manages branding for Firefox.

@jvalleroy, We may need a different name to refer to the box instance.  Currently, we don't have cases where we are referring to box instances (places where "your FreedomBox" or "this FreedomBox" would not suffice). In future, when we encounter them, we can use a different variable for that, I suppose.

@mridulnagpal , @harry-7 we certainly don't want to be troubling our translators with this change.  So, when we change the strings, we need to update the translations too.
